<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1513</width>
    <height>888</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>40</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1513</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStockxx">
    <property name="title">
     <string>Stockxx</string>
    </property>
   </widget>
   <addaction name="menuStockxx"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCursor"/>
   <addaction name="actionLine_Studie"/>
   <addaction name="actionResistance_Studie"/>
   <addaction name="actionFibonacci_Studie"/>
   <addaction name="separator"/>
   <addaction name="actionCross"/>
   <addaction name="actionHand_Mouse"/>
   <addaction name="separator"/>
   <addaction name="actionDelete_All"/>
  </widget>
  <action name="actionLine_Studie">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/LineStudie.png</normaloff>:/imgs/LineStudie.png</iconset>
   </property>
   <property name="text">
    <string>Line Studie</string>
   </property>
  </action>
  <action name="actionResistance_Studie">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/ResStudie.png</normaloff>:/imgs/ResStudie.png</iconset>
   </property>
   <property name="text">
    <string>Resistance Studie</string>
   </property>
  </action>
  <action name="actionCursor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/Cursor.png</normaloff>:/imgs/Cursor.png</iconset>
   </property>
   <property name="text">
    <string>Cursor</string>
   </property>
  </action>
  <action name="actionCross">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/Cross.png</normaloff>:/imgs/Cross.png</iconset>
   </property>
   <property name="text">
    <string>Cross</string>
   </property>
  </action>
  <action name="actionHand_Mouse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/Hand.png</normaloff>:/imgs/Hand.png</iconset>
   </property>
   <property name="text">
    <string>Hand Mouse</string>
   </property>
  </action>
  <action name="actionDelete_All">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/TrashBin.png</normaloff>:/imgs/TrashBin.png</iconset>
   </property>
   <property name="text">
    <string>Delete All</string>
   </property>
  </action>
  <action name="actionFibonacci_Studie">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/Fibonacci.png</normaloff>:/imgs/Fibonacci.png</iconset>
   </property>
   <property name="text">
    <string>Fibonacci Studie</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
